### ISSUE TITLE
Doc guidance: Multiple commands on NFC tag

### ISSUE THREAD

**Issue by johnnyfleet** [ryanolf/node-sonos-nfc#14]
Hi there - apologies if this is the wrong pathway to raise such query. 

I read in the README section under **Card record format**: 

> You can enable cross fade, shuffle, or repeat on a card-by-card basis by adding records to enable these features to the card.

The question I have is, how exactly do I do that? 

What I'm looking to do is clear the queue and decide on a per-card basis whether to enable shuffle etc. 

Is there any information anywhere that helps to provide guidance on how to do this?

Thanks

**Comment by ryanolf** [comment]
Hi @johnnyfleet. You can write multiple NDEF records to a card. Each record can have an instruction. The instruction possibilities are shown [here](https://github.com/hankhank10/vinylemulator). So for example you could have a card that has `spotify:album:4hW2wvP51Myt7UIVTgSp4f` as the first record and `command:shuffle/on` as the second.

**Comment by johnnyfleet** [comment]
Fantastic - that explained it. Apologies for the late reply. I only just attempted it and worked flawlessly. 

For reference this is how it looked in NFC tools. I had to add a next command as well as it always played the same first song - so I wanted to truly have a new song each time I started. 

![image](https://user-images.githubusercontent.com/14134347/172033482-490868b6-9565-47bb-98e2-2050e033d4c1.png)
